### PR TITLE
Xcode 12 warning fixes

### PR DESCRIPTION
**Pull request by ddaddy**
Xcode 12 enables the `CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER` compiler flag which then throws up warnings when compiling.
Simply changing the imports to angled brackets wan't enough as this project uses a different target name between Cocoapods and Carthage.
I have also bumped the minimum deployment target as Xcode 12 also throws warnings when compiling for iOS 8 and MacOS 10.8

Tested using the bundled Example project and also in a personal project that uses Carthage.

### CODE REVIEW

**Comment by Coeur** [comment]
Travis says:
>	Duplicate interface definition for class 'SSZipArchive'

Please address the imports to avoid that issue, thank you.

**Comment by ddaddy** [comment]
The tests needed updating with the Angled brackets. Hopefully this will pass the tests now.

**Comment by ddaddy** [comment]
The Example project has a Carthage target too. It's not used with the tests, but there is a Carthage target.
I was wondering if it might be a good idea to add `s.module_name  = 'ZipArchive'` to the podspec to totally change the module name to match the carthage name of `ZipArchive`? This would likely be a breaking change for people though.

**Comment by Coeur** [comment]
> The Example project has a Carthage target too. It's not used with the tests, but there is a Carthage target.

Yep, I saw it.

> I was wondering if it might be a good idea to add `s.module_name = 'ZipArchive'` to the podspec to totally change the module name to match the carthage name of `ZipArchive`? This would likely be a breaking change for people though.

That question is worth opening a separate issue if there isn't one already.
I believe that such renaming should only be done on a major version upgrade: not worth troubling the users of the current version if we're only doing bug fixes.

**Comment by jhudsonWA** [comment]
I'm not against a name change to match up with GitHub project and Carthage name. But there already is a cocoa pod with the ZipArchive name and it is not ours: https://cocoapods.org/pods/ZipArchive

**Comment by ddaddy** [comment]
Ah that's a problem then. Maybe it's worth looking to do it the other way and have Carthage build as SSZipArchive

**Review by Coeur — APPROVED** [review]
Approved, but in the changelog, let's mention about the minimum macOS version change from 10.8 to 10.9.

File: Example/Podfile [hunk 1]
```diff
@@ -16,7 +16,7 @@ abstract_target 'core' do
   end
 
   target 'ObjectiveCExampleTests_macOS' do
-    platform :osx, '10.8'
+    platform :osx, '10.9'
```
Comment (Coeur) [inline Example/Podfile:19]: Let's change it in the `SSZipArchive.podspec` too: we're not maintaining 10.8 anymore.

Comment (ddaddy) [reply Example/Podfile:19]: I'm sure I changed that. Are you looking at the latest Fixed Example Tests commit?

Comment (Coeur) [reply Example/Podfile:19]: Oh, I missed it.

**Comment by ddaddy** [comment]
I've updated the Readme to reflect the new MacOS 10.9 minimum

**Comment by Coeur** [comment]
Thank you Wilson.
Well, good to merge once Travis build is done.

**Comment by willson-chen** [comment]
@Coeur Yes, you are right, the travis-ci document lists the detailed information [OSX](https://docs.travis-ci.com/user/reference/osx/). Thank you very much, I have modified it.

**Pull request merged**